src/main/java/it/gov/pagopa/admissibility/controller/AdmissibilityController.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.admissibility.controller;
+
+import it.gov.pagopa.admissibility.dto.onboarding.InitiativeStatusDTO;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import reactor.core.publisher.Flux;
+
+/**
+ * Component that exposes APIs
+ * */
+@RequestMapping("/idpay/admissibility")
+public interface AdmissibilityController {
+
+    @GetMapping(value = "/initiative/{initiativeId}")
+    Flux<InitiativeStatusDTO> getInitiativeStatus(@PathVariable("initiativeId") String initiativeId);
```

Comment:
why a Flux? given an initiativeId we expect to retrieve at most one result, thus a Mono